### PR TITLE
Bug/chat input

### PR DESCRIPTION
**Pull request by vkrm-segta**

### CODE REVIEW

File: packages/ui/src/views/chatmessage/ChatMessage.js [hunk 1]
```diff
@@ -366,6 +366,7 @@ export const ChatMessage = ({ open, chatflowid, isDialog }) => {
                             value={userInput}
                             onChange={onChange}
                             multiline={true}
+                            maxRows={2}
```
Comment (HenryHengZJ) [inline packages/ui/src/views/chatmessage/ChatMessage.js:369]: Prefer `maxRows={isDialog ? 7 : 2}`

**Comment by HenryHengZJ** [comment]
also looks like you have already signed CLA: 
image

File: packages/ui/src/views/chatmessage/ChatMessage.js (hunk 1 above)
Comment (vkrm-segta) [reply packages/ui/src/views/chatmessage/ChatMessage.js]: Sure

File: packages/ui/src/views/chatmessage/ChatExpandDialog.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ const ChatExpandDialog = ({ show, dialogProps, onClear, onCancel }) => {
                     )}
                 </div>
             </DialogTitle>
-            <DialogContent>
+            <DialogContent sx={{ display: 'flex', justifyContent: 'flex-end', flexDirection: 'column' }}>
```
Comment (HenryHengZJ) [inline packages/ui/src/views/chatmessage/ChatExpandDialog.js:46]: tried this and didnt see any changes on ui, whats this for?

Comment (vkrm-segta) [reply packages/ui/src/views/chatmessage/ChatExpandDialog.js:46]: This happen when i paste text into input without this code

https://github.com/FlowiseAI/Flowise/assets/124426279/c504a2a1-de6c-4cd7-b37a-3bffe511b5b4

Comment (vkrm-segta) [reply packages/ui/src/views/chatmessage/ChatExpandDialog.js:46]: I can fix  this.

**Comment by vkrm-segta** [comment]
@HenryHengZJ fixed the issue for hide text underneath,  when you paste long text in dialog.

**Review by HenryHengZJ — APPROVED** [review]
thanks @vkrm-segta !

**Pull request merged**